config/channel/distributed/500-controller-deployment.yaml

```diff
@@ -70,9 +70,9 @@ spec:
             cpu: 20m
             memory: 25Mi
         volumeMounts:
-          - name: config-kafka
-            mountPath: /etc/config-kafka
-      volumes:
         - name: config-kafka
```

Comment:
the indent is off (but still correct) thinking there is a change when there is not?

Reply:
ah... intellij.... sigh... - I'll correct it.   pretty sure it works either way but good to stay consistent

Reply:
nvm, it's consistent with the rest